README.md

```diff
@@ -1 +1,61 @@
-# Main-mobile1
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portolio-Project
+
+> Description the project.
+  
+  This a Portolio webpage with a header, headline section, work and about section for mobile versions.
+  
+  ![Screenshot (10)](https://user-images.githubusercontent.com/103378039/192074078-86413b38-0a4c-4b41-968f-891d96c4c363.png)
+
+  ![Screenshot (8)](https://user-images.githubusercontent.com/103378039/192074086-5eda030b-ff72-4924-8798-f391380ff570.png)
+  
+  ![Screenshot (9)](https://user-images.githubusercontent.com/103378039/192074090-c53283e0-df81-4ded-8f57-bc94fee683b2.png)
+
+## Built With
+
+- HTML
+- CSS
+
+
+## Getting Started
+
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+  - Git must be installed on local computer
+
+### Setup
+  - git clone https://github.com/mutesihope/Main-mobile.git
+  
+  - cd My-Portfolio
+
+
+## Authors
+
+- GitHub: [@githubhandle](https://github.com/mutesihope)
+- Twitter: [@twitterhandle](https://twitter.com/KarangwaMutesi)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/karangwa-mutesi-hope)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] **[OPTIONAL]** Please 🙏🏽 make sure that all the links in the README file work as expected. This link doesn't work because you haven't created the LICENSE file in your repository 📑 👀